deep_gemm/__init__.py

```diff
@@ -1,15 +1,29 @@
-import torch
-
-from . import jit
-from .jit_kernels import (
-    gemm_fp8_fp8_bf16_nt,
-    m_grouped_gemm_fp8_fp8_bf16_nt_contiguous,
-    m_grouped_gemm_fp8_fp8_bf16_nt_masked,
-    wgrad_gemm_fp8_fp8_fp32_nt,
-    k_grouped_wgrad_gemm_fp8_fp8_fp32_nt,
-    ceil_div,
-    set_num_sms, get_num_sms,
-    get_col_major_tma_aligned_tensor,
```

Comment:
`get_col_major_tma_aligned_tensor` is this function deprecated?

Reply:
Also, if we don't import the function here, the users have to refactor the code. Eg:
```bash
from deep_gemm import calc_diff, ceil_div

->

from deep_gemm.testing.numeric import calc_diff
from deep_gemm.utils.math import ceil_div
```
Is this expected?

Reply:
Make sense to me, thanks!

Reply:
@RayWang96 When you said not used in SM100, does this mean it will be automatically handled in SM100 and we don't need the code like https://github.com/vllm-project/vllm/blob/dac8cc49f43f7d2639d873532a408949169821a9/vllm/model_executor/layers/quantization/fp8.py#L654?

Reply:
So do you mean it is recommended that we call this `get_col_major_tma_aligned_tensor` each time before we call any gemm operation?